draft-ietf-lake-edhoc.md

```diff
@@ -585,7 +585,11 @@ ead = 1* (
 )
 ~~~~~~~~~~~
 
-A security application using external authorization data need to register an ead_label, specify the ead_value format for each message (see {{iana-ead}}), and describe processing and security considerations.
+A security application using external authorization data need to register an ead_label, specify the ead_value formatting for each message (see {{iana-ead}}), and describe processing and security considerations.
+
+An EAD field can be either critical or non-critical, determined by the sign of ead_label: negative means critical; non-negative means non-critical. If an endpoint does not support a received critical EAD field, the EDHOC protocol MUST be discontinued.
```

Comment:
"If an endpoint does not support a received critical EAD field, the EDHOC protocol MUST be discontinued."

This needs more discussion and work. The handling of critical extensions in C.509 is to my understanding confusing and problematic. Both in the  specification and in implementations.

What does support means. Can an endpoint supporting the EAD ignore it.